src/models/Stall.ts

```diff
@@ -127,7 +127,7 @@ class Stall extends Model<StallAttributes, StallCreationAttributes> implements S
   public removeFavourites!: HasManyRemoveAssociationsMixin<Favourite, number>;
   public setFavourites!: HasManySetAssociationsMixin<Favourite, number>;
 
-  public readonly HawkerCentre?: HawkerCentre;
+  public readonly HawkerCentre!: HawkerCentre;
```

Comment:
I think we should still keep the `HawkerCentre` optional here because that is the default given by the Sequelize library.